### DISCUSSION TITLE
How to use NVDXInterop extension?

### DISCUSSION THREAD

**Discussion by qian-o** [dotnet/Silk.NET#1466] [Q&A] [answered]
Hello, I am trying to use the WGL context attribute to create NVDXInterop, but I am getting a lot of loading exceptions. I am not a professional OpenGL developer and I am not sure how to use the WGL extension correctly. Could you provide a code example?
![image](https://github.com/dotnet/Silk.NET/assets/84434846/45005e35-d6a7-4767-8aef-9100316f2491)

**Comment by Perksey** [comment]
```cs
WGL wgl = WGL.GetApi();
if (!wgl.TryGetExtension(out NVDXInterop dxInterop))
{
    throw new("DX interop extension missing");
}
```

**Reply by Perksey** [reply, marked as answer]
Oh, it would appear that our WGL bindings are not implemented yet and are not ready for use. Apologies for this, please do not use these bindings.

**Reply by qian-o** [reply]
I tried implementing the WGL extension using the Glfw.GetProcAddress function within the LamdaNativeContext class, and it seems to be working fine. Thank you for your help and your answer, thank you.
![image](https://github.com/dotnet/Silk.NET/assets/84434846/ef9dac2f-ba1c-40d5-bbbe-29da860312e3)

**Reply by Perksey** [reply]
Excellent, but please note this is an unsafe and unsupported way of using the library. We will strive to provide a safe & supported way in the future.
